docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1517,6 +1603,133 @@ rpc-http-port="3435"
 Specifies HTTP JSON-RPC listening port (TCP).
 The default is 8545. Ports must be [exposed appropriately](../../HowTo/Find-and-Connect/Configuring-Ports.md). 
 
+### rpc-http-tls-ca-clients-enabled
+
+```bash tab="Syntax"
+--rpc-http-tls-ca-clients-enabled
+```
+
+```bash tab="Environment Variable"
+BESU_RPC_HTTP_TLS_CA_CLIENTS_ENABLED[=<true|false>]
```

Comment:
To be consistent, I think this needs to be an example: 
`BESU_RPC_HTTP_TLS_CA_CLIENTS_ENABLED=true`

Reply:
Done. Mistakenly copied syntax example to Env Variable section.